one-ui/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.test.tsx

```diff
@@ -13,6 +13,8 @@ describe('Source to entity mapping dialog component', () => {
     expect(wrapper.find('#srcContainer').length).toEqual(1)
     expect(wrapper.find('#srcDetails').length).toEqual(1)
     expect(wrapper.find('#entityContainer').length).toEqual(1)
+    expect(wrapper.find('#noData').length).toEqual(1)
+    expect(wrapper.find('#dataPresent').length).toEqual(0)
     expect(wrapper.find('#successMessage').length).toEqual(1)
     expect(wrapper.find('#errorMessage').length).toEqual(1)
     expect(wrapper.find('#listIcon').length).toEqual(1)
```

Comment:
can you pls try running npm run test.. we dont show successMessage, errorMessage and listIcon in the DOM yet and am sure they will fail.